layers/+misc/multiple-cursors/README.org

```diff
@@ -60,3 +59,43 @@ For easy navigation you also have the following:
 | ~C-t~       | evil-mc-skip-and-goto-next-match   |
 | ~C-M-j~     | evil-mc-make-cursor-move-next-line |
 | ~C-M-k~     | evil-mc-make-cursor-move-prev-line |
+|             |                                    |
+** =multiple-cursors=
+The =multiple-cursors= backend provides the following key bindings to
+insert new cursors:
+
+| Key binding | Description                       |
+|-------------+-----------------------------------|
+| ~g r r~     | mc/mark-more-like-this-extended   |
+| ~g r a~     | mc/mark-all-like-this             |
+| ~g r d~     | mc/mark-all-like-this-in-defun    |
+| ~g r l~     | mc/edit-lines                     |
+
+These special bindings manipulate text under cursors:
+
+| Key binding | Description                   |
+|-------------+-------------------------------|
+| ~g r s s~   | mc/sort-regions               |
+| ~g r s r~   | set-rectangular-region-anchor |
+| ~g r s m~   | mc/mark-sgml-tag-pair         |
+| ~g r s n~   | mc/insert-numbers             |
+| ~g r s l~   | mc/insert-letters             |
+| ~g r s t~   | mc/reverse-regions            |
+* Notes
+** =multiple-cursors=
+Some commands executed during =multiple-cursors= enabled may lead Emacs to go frenzy.
+Commands like window manipulation will be executed multiple times if =mc= is active.
+Not all Emacs commands may be compatible with =mc=.
+
+To run interactive ~M-x~ commands with =mc=, run the command first. This will result
+with application of the command to the leading cursor. Then press =C-:= to apply the
+command to consequtive cursors. If this is not the case or you want a command to be
+executed only once, configure the =mc/cmds-to-run-once= variable of the layer like
+in the following example.
+
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(
```

Comment:
Maybe the first line should be excluded:
`(setq-default dotspacemacs-configuration-layers '(`
(and it's closing parenthesis)

Because there has been questions in the gitter chat about people trying to add the whole code block to their user-config section and running into issues.

There could possibly be a mention above that the code segment, that it should be added inside the 
`dotspacemacs-configuration-layers` list, near the top of `.spacemacs`.

And the indentation of the command should probably match how `elisp-mode` indents them:
```
(multiple-cursors :variables
                  multiple-cursors-backend 'mc
                  mc/cmds-to-run-once '(upcase-region))
```

Reply:
Fixed the indentation.

Reply:
Yeah. If `dotspacemacs-configuration-layers` in the code blocks it's going to be changed, then it should be changed in separate PR that changes it everywhere.